setting-up-tugboat/index.md

```diff
@@ -0,0 +1,510 @@
+# Setting up your Tugboat
+
+- [Connect with your provider](#connect-with-your-provider)
+- [Create a new project](#create-a-new-project)
+- [Add repos to the project](#add-repos-to-the-project)
+- [Select repo settings (optional)](#repository-settings-optional)
+- [Create a Tugboat Config file](#create-a-tugboat-config-file)
+
+> #### Hint::Video Overview
+>
+> If you want to learn more about setting up Tugboat and get a feel for the
+> overall workflow, watch our
+> [Getting Started with Tugboat video](https://www.youtube.com/watch?v=HYTsrm5ORmU)
+> on YouTube.
+
+## Connect with your provider
+
+- [Generic git server](#generic-git-server)
+- [GitHub](#github)
+- [GitLab](#gitlab)
+- [BitBucket](#bitbucket)
+- [Add a link to another git provider](#adding-a-link-to-a-git-provider)
+
+### Generic Git Server
+
+If you're not using GitHub, GitLab, or BitBucket, you can use a generic git
+server with Tugboat. You'll [sign in to Tugboat](https://dashboard.tugboat.qa/)
+using an email address, and when you go to add a repo to your Tugboat project,
+you can link to a Git URL.
+
+> #### Warning::Authenticate with a git provider to use auto-build Preview functionality
```

Comment:
If Generic Git is last, we could probably just incorporate this into the text without a call-out